### PR TITLE
Fixing some commentaries that were meant to be code

### PR DESCRIPTION
**Pull request by ogabriel**
the codes of creation of the structs were like comments

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @ogabriel! Will you bump the lesson version from `1.4.0` to `1.5.0`?

**Comment by ogabriel** [comment]
@doomspork i updated here, but i checked the other languages, and all of them seems to have the same problem, so i will fix them too

**Comment by ogabriel** [comment]
fixed in all languages

**Comment by lex111** [comment]
> @doomspork i updated here, but i checked the other languages, and all of them seems to have the same problem, so i will fix them too
> fixed in all languages


Oh no, it definitely doesn’t have to be done! Basause then it will seem that the  `baiscs/module` file contains *up-to-date* information, although this is not the case (depending on the language)!

**Comment by ogabriel** [comment]
ok @lex111 , i have undone the commit

**Comment by lex111** [comment]
We probably did **not bump** the version in the translations in this case (it's not updating the information, and in fact the _fix bug_), what do you think @doomspork?

**Comment by doomspork** [comment]
@lex111 we do want to bump the version if it's a change that needs to be applied to all the lessons. The versioning rules (MAJOR.MINOR.PATCH) aren't very clear because there's some fluidity with translations.

As someone with as much experience as you have in translations, I would _love_ your thoughts on how we could handle this better.

**Comment by lex111** [comment]
@doomspork I think in this case, everything is quite simple. This is **not** an update of information (that is, does not apply to the text, which means that the translation is not needed), so I would update this lesson everywhere, and did **not bump** the version (including the English). 
Although, ideally, of course, to bump the version, but there is a slightly different model (?) of lesson versioning, which, as I understand it, concerns the information itself (adding, changing, deleting).

**Comment by ogabriel** [comment]
@doomspork @lex111 hello again guys, i really didnt get what are you saying. What should i do what now?

**Comment by lex111** [comment]
@ogabriel sorry for the delay! @doomspork please help us, what the final decision?

**Comment by gemantzu** [comment]
@lex111 I agree with this, we should fix it over all languages and _not_ bump the version anywhere.

**Comment by doomspork** [comment]
Sorry @ogabriel / @lex111, it's been a very hectic week. What @gemantzu and you said sounds good. Let's _not_ bump the version and apply this change to all the languages.

@ogabriel I apologizes for this taking so long. Would you prefer I merge this and take care of those changes?

**Comment by ogabriel** [comment]
@doomspork i still have te commit with the changes on all languages in another branch of mine, i can just cherry pick it, and remove the bump in all the versions

But its your call

**Comment by doomspork** [comment]
Wonderful! If you still have the commit let's do that @ogabriel 😁 I didn't want to inconvenience you any further.

**Comment by ogabriel** [comment]
@doomspork i did it

there are some trailing whitespaces that my editor removed too, but i think that isnt a problem

**Comment by julienXX** [comment]
Shouldn't `mananger` be `manager`?

File: bn/lessons/basics/modules.md [hunk 1]
```diff
@@ -82,13 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
```
Comment (doomspork) [inline bn/lessons/basics/modules.md]: Thank you @ogabriel! One small change. I think this should be `manager` (with only 1 `n`).

File: de/lessons/basics/modules.md [hunk 2]
```diff
@@ -80,14 +80,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline de/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: en/lessons/basics/modules.md [hunk 3]
```diff
@@ -88,14 +88,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline en/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: es/lessons/basics/modules.md [hunk 4]
```diff
@@ -85,14 +85,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline es/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: fr/lessons/basics/modules.md [hunk 5]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline fr/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: gr/lessons/basics/modules.md [hunk 6]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline gr/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: id/lessons/basics/modules.md [hunk 7]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline id/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: it/lessons/basics/modules.md [hunk 8]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline it/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: ja/lessons/basics/modules.md [hunk 9]
```diff
@@ -88,14 +88,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline ja/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: ko/lessons/basics/modules.md [hunk 10]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline ko/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: ms/lessons/basics/modules.md [hunk 11]
```diff
@@ -82,20 +82,20 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline ms/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: no/lessons/basics/modules.md [hunk 12]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline no/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: pl/lessons/basics/modules.md [hunk 13]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline pl/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: pt/lessons/basics/modules.md [hunk 14]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline pt/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: ru/lessons/basics/modules.md [hunk 15]
```diff
@@ -87,14 +87,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline ru/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: sk/lessons/basics/modules.md [hunk 16]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline sk/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: tr/lessons/basics/modules.md [hunk 17]
```diff
@@ -81,14 +81,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline tr/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: uk/lessons/basics/modules.md [hunk 18]
```diff
@@ -82,14 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline uk/lessons/basics/modules.md]: typo here `mananger` should be `manager`

File: vi/lessons/basics/modules.md [hunk 19]
```diff
@@ -77,22 +77,21 @@ end
...
 iex> %Example.User{}
 #Example.User<name: "Sean", roles: [], ...>
 
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
+#Example.User<name: "Steve", roles: [:mananger]>
```
Comment (tajchumber) [inline vi/lessons/basics/modules.md]: typo here `mananger` should be `manager`

**Comment by ogabriel** [comment]
hi @tajchumber @doomspork
sorry for the typo, i just copied through the files and really didnt notice it

and i found 2 other languages that i didnt fixed the code, and i fixed it too

**Comment by doomspork** [comment]
Thank you _so much_ @ogabriel! Sorry this PR took a little bit to get merged but I'm really grateful for your help. Please do stick around 😁

**Comment by ogabriel** [comment]
Thank you all

I intend to stick around, this project is helping me alot to learn a new language.

And i am glad to contribute 😄

**Pull request merged**